src/librustdoc/html/render.rs

```diff
@@ -65,12 +65,10 @@ use html::item_type::ItemType;
 use html::layout;
 use html::markdown::Markdown;
 use html::markdown;
-use html::escape::Escape;
 use stability_summary;
 
 /// A pair of name and its optional document.
-#[derive(Clone, Eq, Ord, PartialEq, PartialOrd)]
-pub struct NameDoc(String, Option<String>);
+pub type NameDoc = (String, Option<String>);
```

Comment:
The auto-serialization of the struct is much longer than that of an anonymous tuple: `{"_field0":...,"_field1":...}` I think.